CONTRIBUTING.md

```diff
@@ -39,6 +39,38 @@ request into the body of the commit message.
 
 If the Travis tests are failing on your branch, you should look at the logs to figure out why. Sometimes they fail spuriously, in which case you can post a comment requesting that a project owner kick the build.
 
+# Error handling
+
+All errors must be addressed in some way: That may be simply by returning an
+error up the stack, or by handling it in some intelligent way where it is
+generated, or explicitly ignored by assigning it to `_`. We use the `errcheck`
+tool in our integration tests to make sure all errors are addressed. Note that
+even in tests, errors should not be ignored, since they may generate
```

Comment:
I think I know what you're saying but this seems confusing since we just explained how to ignore an error by assigning to `_`. Maybe something like "Ignoring errors should be a last resort, even in tests, since it may generate hard-to-debug problems"?